cdap-tms/src/main/java/co/cask/cdap/messaging/store/TableFactory.java

```diff
@@ -28,4 +28,6 @@
   MessageTable createMessageTable(String tableName) throws IOException;
 
   PayloadTable createPayloadTable(String tableName) throws IOException;
+
+  void upgradeTables(String messageTableName, String payloadTableName) throws IOException;
```

Comment:
Better have one `tableName`. The reason is for supporting topic to table mapping in future.